packages/gatsby/src/commands/build-html.ts

```diff
@@ -35,15 +41,19 @@ const doBuildRenderer = async (program, webpackConfig) => {
   return outputFile
 }
 
-const buildRenderer = async (program, stage, { parentSpan }) => {
+const buildRenderer = async (
+  program: IProgram,
+  stage: BuildHTMLStage,
+  parentSpan: IActivity
```

Comment:
`parentSpan` is destructured in the previous code. Was that a mistake? It does seem a bit odd to destructure it then immediately recreate the object below.

Reply:
https://gatsbyjs.slack.com/archives/CA1GW1HNU/p1579121474098900